packages/vaadin-themable-mixin/src/lumo-injector.js

```diff
@@ -77,6 +77,9 @@ export class LumoInjector {
   /** @type {Map<string, CSSStyleSheet>} */
   #styleSheetsByTag = new Map();
 
+  /** @type {Map<string, Set<HTMLElement>>} */
+  #componentsByTag = new Map();
```

Comment:
nit: the property name was not clear to me at first, that it stores instances of the components by tag name.

Reply:
Could you suggest an improvement?